### PR TITLE
feature: Add static defines of notification strings

### PR DESCRIPTION
**Pull request by kirre-bylund**

### CODE REVIEW

**Review by ladydascalie — APPROVED** [review]

File: Runtime/Game/Requests/NotificationRequests.cs [hunk 1]
```diff
@@ -28,6 +28,85 @@ public enum LootLockerNotificationContentKind
...
+    public struct LootLockerStandardContextKeys
+    {
+        /// <summary>
+        /// Standard context keys to expect when source is triggers
+        /// </summary>
+        public struct Triggers
+        {
+            public static readonly string TriggerID = "trigger_id";
+            public static readonly string TriggerKey = "trigger_key";
+            public static readonly string TriggerLimit = "trigger_limit";
```
Comment (JohannesLoot) [inline Runtime/Game/Requests/NotificationRequests.cs:68]: Maybe rename these to just id, key and limit? It is specified in the struct above that it is Triggers.

**Review by JohannesLoot — APPROVED** [review]
Code checks out!
Solid work 👍 
Left one comment, fine by me to merge if you don't agree.

File: Runtime/Game/Requests/NotificationRequests.cs (hunk 1 above)
Comment (kirre-bylund) [reply Runtime/Game/Requests/NotificationRequests.cs:68]: SMORT

Comment (kirre-bylund) [reply Runtime/Game/Requests/NotificationRequests.cs]: Fixed

**Pull request merged**